client/webserver/locales/en-us.go

```diff
@@ -63,14 +63,15 @@ var EnUS = map[string]string{
 	"Market Order":                   "Market Order",
 	"reg_status_msg":                 `In order to trade at <span id="regStatusDex" class="text-break"></span>, the registration fee payment needs <span id="confReq"></span> confirmations.`,
 	"Buy":                            "Buy",
-	"Sell":                           "Sell",
+	"buy_switch":                     "⇄ Buy",
+	"sell_switch":                    "⇄ Sell",
 	"lot_size":                       "Lot Size",
 	"Rate Step":                      "Rate Step",
 	"Max":                            "Max",
 	"lot":                            "lot",
 	"min trade is about":             "min trade is about",
 	"immediate_explanation":          "If the order doesn't fully match during the next match cycle, any unmatched quantity will not be booked or matched again. Taker-only order.",
-	"Immediate or cancel":            "Immediate or cancel",
+	"Immediate only":                 "Immediate only",
```

Comment:
This seems to be slightly more accurate, since "Immediate or cancel" feels like whole order will be canceled if something didn't match, as opposed to "only immediate part" will be filled. 

And, short text looks better in UI.